corehq/ex-submodules/couchexport/models.py

```diff
@@ -141,12 +141,14 @@ def last(cls, index):
 
     @classmethod
     def get_all_checkpoints(cls, index):
-        return cls.view("couchexport/schema_checkpoints",
+        doc_ids = [result["id"] for result in cls.get_db().view(
+            "couchexport/schema_checkpoints",
             startkey=[json.dumps(index)],
             endkey=[json.dumps(index), {}],
-            include_docs=True,
             reduce=False,
-        )
+        )]
+        for doc in iter_docs(cls.get_db(), doc_ids):
+            yield cls.wrap(doc)
```

Comment:
It's not clear to me how this is different from before, except now it's doing at least two couch queries where previously it did one. However, it's still loading all docs into memory (although not if the generator is not fully listed), correct? How is this better?